geode-core/src/main/java/org/apache/geode/security/AuthInitialize.java

```diff
@@ -36,28 +33,31 @@
  */
 public interface AuthInitialize extends CacheCallback {
 
+  public static String SECURITY_USERNAME = "security-username";
+  public static String SECURITY_PASSWORD = "security-password";
+
   /**
    * Initialize the callback for a client/peer. This is invoked when a new connection from a
    * client/peer is created with the host.
    *
+   * Do not use these loggers, use log4j logger directly
+   *
    * @param systemLogger {@link LogWriter} for system logs
    * @param securityLogger {@link LogWriter} for security logs
    *
    * @throws AuthenticationFailedException if some exception occurs during the initialization
    *
-   * @deprecated since Geode 1.0, use init()
    */
-  @Deprecated
-  void init(LogWriter systemLogger, LogWriter securityLogger) throws AuthenticationFailedException;
+  default void init(LogWriter systemLogger, LogWriter securityLogger)
+      throws AuthenticationFailedException {};
 
   /**
-   * @since Geode 1.0. implement this method instead of init with logwriters. Implementation should
-   *        use log4j instead of these loggers.
+   *
+   * @since Geode 1.0.
+   * @deprecated in Geode 1.5, never called by the product, use init(LogWriter systemLogger,
```

Comment:
Linking `init` could be nice.  I'd prefer full sentences in the javadocs personally, rather that notes.